addons/mail/static/src/models/thread/thread.js

```diff
@@ -636,6 +636,7 @@ registerModel({
                 attachments: attachmentsData,
                 followers: followersData,
                 suggestedRecipients: suggestedRecipientsData,
+                hasAccess,
```

Comment:
guideline: alphabetical order for object keys

Reply:
> guideline: alphabetical order for object keys

Done.Thanks